src/components/pickers/PickerDrawer.tsx

```diff
@@ -9,23 +9,45 @@ import {
   Select,
   Typography,
 } from "@mui/material";
-import { BloodType, IPicker } from "project-2-types/lib/pickers";
+import { createPicker } from "api/pickers";
+import { BloodType, IPicker, Relationship } from "project-2-types/lib/pickers";
 import { useState } from "react";
 import { Controller, useForm } from "react-hook-form";
+import { useMutation, useQueryClient } from "react-query";
 
 interface IPickerForm extends Omit<IPicker, "id"> {}
 
+export const relationshipList = (
+  Object.keys(Relationship) as Array<keyof typeof Relationship>
+).map((key) => ({ value: key, label: Relationship[key] }));
+
+export const bloodTypeList = (
+  Object.keys(BloodType) as Array<keyof typeof BloodType>
+).map((key) => ({ value: key, label: BloodType[key] }));
+
 const PickerDrawer = () => {
```

Comment:
These two input types should just be swapped:
![image](https://github.com/LangaraVoyagers/project-2/assets/89278135/a92f1e71-bb55-43ef-82ea-43a47a8275b9)

Reply:
<img width="468" alt="Screenshot 2024-02-22 at 3 41 53 PM" src="https://github.com/LangaraVoyagers/project-2/assets/57273227/4b0c100a-61d0-4ef9-8a10-c5dc2e6af2dc">